app/assets/stylesheets/scaffolds.scss

```diff
@@ -87,11 +87,7 @@ body.ten-x-hacker-theme {
   align-self: center;
 }
 
-.trusted-visible-block {
```

Comment:
I think this is causing a logical error where the shield now is _always_ visible in wide screen (instead of just for `trusted`)...

It looks like the `Moderation Center` link is also now always present as opposed to just there for trusted users.

Reply:
Ah I think you're right. Let me adjust.